components/dear/sources/sale-shipment-tracking-number-changed/sale-shipment-tracking-number-changed.mjs

```diff
@@ -0,0 +1,32 @@
+import constants from "../../common/constants.mjs";
+import base from "../common/webhooks.mjs";
+
+export default {
+  ...base,
+  name: "Shipment Tracking Number Change",
+  key: "dear-sale-shipment-tracking-number-change",
+  type: "source",
+  description: "Emit new event when a shipment tracking number changes.",
```

Comment:
```suggestion
  description: "Emit a new event when a shipment tracking number changes.",
```

Reply:
Changed "Emit new event" to "Emit a new event" to follow Google's style guide on articles.